app/frontend/src/components/UI/sprite/Sprite.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-webpack-loader-syntax */
+import React from 'react';
+
+/**
+ * Icons must be imported separately because some of them may not be used in application
+ * otherwise try to use following syntax:
+ */
+// eslint-disable-next-line import/no-webpack-loader-syntax
+// import icons from '!!raw-loader!icons/*.svg';
+
+import HomeIcon from '!!raw-loader!icons/home.svg';
+import DefaultAvatar from '!!raw-loader!icons/DefaultAvatar.svg';
+import PlusIcon from '!!raw-loader!icons/plus.svg';
+import SettingsIcon from '!!raw-loader!icons/settings.svg';
+
+const icons = {
+  'home': HomeIcon,
+  'DefaultAvatar': DefaultAvatar,
+  'plus': PlusIcon,
+  'settings': SettingsIcon,
+};
+
+export const viewBoxes = new Map();
+
+const getViewBox = (xml:string):string => {
+  const doc = new DOMParser().parseFromString(xml, 'text/html');
+
+  return doc?.querySelector('svg')?.getAttribute('viewBox') || '';
+};
+
+const xmlns = 'http://www.w3.org/2000/svg';
+
+const html = Object.entries(icons)
+  .map(([name, icon]) => {
+    viewBoxes.set(name, getViewBox(icon));
+
+    return icon
+      .replace(`xmlns="${xmlns}"`, `id="sprite-${name}"`)
+      .replaceAll(/\bsvg\b/g, 'symbol');
+  })
+  .join('');
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Does the next line really violate naming convention? if not, i would remove this comment

Reply:
> Error:   43:96  error  Object Literal Property name `__html` trimmed as `_html` must match one of the following formats: camelCase, PascalCase, UPPER_CASE  @typescript-eslint/naming-convention

Reply:
There is always another solution where can not use dangerouslySetInnerHTML property - write build task for generating sprite file separately.